features/attached_enable.feature

```diff
@@ -79,13 +90,23 @@ Feature: Enable command behaviour when attached to an UA subscription
             Cannot enable unknown service 'foobar, ros'.
             Try cc-eal, cis, esm-infra, fips, fips-updates, livepatch.
             """
+        And I verify that running `ua enable ros foobar --format json` `with sudo` exits `1`
+        And I will see the following on stdout:
+        """
+        {"_schema_version": 0.1, "errors": [{"message": "Cannot enable unknown service 'foobar, ros'.\nTry cc-eal, cis, esm-infra, fips, fips-updates, livepatch.", "service": null, "type": "system"}], "failed_services": ["foobar", "ros"], "needs_reboot": false, "processed_services": [], "result": "failure", "warnings": []}
+        """
         And I verify that running `ua enable esm-infra` `with sudo` exits `1`
-        Then I will see the following on stdout:
+        And I will see the following on stdout:
             """
             One moment, checking your subscription first
             UA Infra: ESM is already enabled.
             See: sudo ua status
             """
+        And I verify that running `ua enable esm-infra --format json` `with sudo` exits `1`
+        Then I will see the following on stdout:
+            """
+            {"_schema_version": 0.1, "errors": [{"message": "UA Infra: ESM is already enabled.\nSee: sudo ua status", "service": "esm-infra", "type": "service"}], "failed_services": ["esm-infra"], "needs_reboot": false, "processed_services": [], "result": "failure", "warnings": []}
+            """
```

Comment:
We should probably have some more integration tests. Some scenarios might be:
* successful single service enable
* successful multiple service enable
* multiple service, one success, one failure
* something that results in `needs_reboot: true`